rpc/src/client/transport/http.rs

```diff
@@ -367,8 +367,21 @@ mod sealed {
         {
             let request = self.build_request(request)?;
             let response = self.inner.request(request).await.map_err(Error::hyper)?;
+
+            tracing::debug!("Incoming response:");
+            let response_status = response.status();
+            tracing::debug!("  Status: {}", response.status());
             let response_body = response_to_string(response).await?;
-            tracing::debug!("Incoming response: {}", response_body);
+            tracing::debug!("  Body: {}", response_body);
```

Comment:
I'd rather put out a single tracing event, using keywords:

```rust
    tracing::debug!(status = %response_status, body = %response_body, "incoming response");
```

On the other hand, is it too much spam for debug level? Maybe the body dump should be on `trace` level.

Reply:
Yeah let's do that. I think it's fine at the debug level, and we're using it as via from Hermes `--debug=rpc` already, so let's leave it like that.

Reply:
e64df0b3